src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -52,6 +52,7 @@ public ReleaseConfiguration() {
     //TODO currently it's not clear when to use class fields and when to use the 'configuration' map
     //Let's make it clear in the docs
     private boolean dryRun = true;
+    private boolean releaseAllJavaModules = true;
```

Comment:
Team, are you ok with this name of the property? It's a public API so let's choose the best name :) Some alternatives:

- shipkit.releaseAllJavaModules
- shipkit.releaseAllJavaSubprojects
 - shipkit.publishAllJava
 - shipkit.publishEveryJavaSubproject
 - shipkit.shipEveryJavaModule
 - shipkit.java.shipAll